src/FactSystem/ParameterManager.cc

```diff
@@ -19,6 +19,7 @@
 #include "CompInfoParam.h"
 #include "FTPManager.h"
 
+
```

Comment:
Not necessary?

Reply:
No. In one of my previus commits (09b6219f41ed370848da3dede5a758b86049dca7) in the line 260 i write the logic to show the autoload checkbox so i need to #include SettingManager.h to acces to the settings. Then i change of place were to write it because here it wasn't the place.
the logic was remeved too but it doest show in git idk why. Check the line 260 to verify

Reply:
Yes. I just meant since the change got removed you can delete this empty line (line 22). There is no need to change the file anymore.

Reply:
ok